lib/utilities/strings.rb

```diff
@@ -58,9 +59,18 @@ def self.termify(string)
   # @return [Boolean]
   #   whether the string is an integer (positive or negative)
   # see http://stackoverflow.com/questions/1235863/test-if-a-string-is-basically-an-integer-in-quotes-using-ruby
+  # Note:  Might checkout CSV::Converters constants to see how they handle this
   def self.is_i?(string)
     /\A[-+]?\d+\z/ === string
   end
 
+  # @return [String, param]
+  #   the goal is to sanitizie an individual string such that it is usable in *TAB* delimited, UTF-8, column.  See Download
+  #   TODO: Likely need to handle quotes
+  def self.sanitize_for_csv(string)
+    return string if string.blank?
+    string.to_s.gsub(/\n/, '\n').gsub(/\t/, '\t')
```

Comment:
Good points.  I was thinking round tripping perhaps, but you're right.  If we do use markup we should pick something like Markdown.  Perhaps make it optional.